src/PrestaShopBundle/Form/Admin/Type/SearchAndResetFormType.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Form\Admin\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\ResetType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * FormType used in rendering of "Search and Reset" action in Grids.
+ */
+class SearchAndResetFormType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('search', SubmitType::class, [
+                'label' => 'Search',
+                'attr' => [
+                    'class' => 'btn btn-primary',
+                ],
+            ])
+            ->add('reset', ResetType::class, [
+                'label' => 'Reset',
+                'attr' => [
+                    'class' => 'btn btn-link reset-search',
+                ],
+            ])
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'translation_domain' => 'Global.Actions',
```

Comment:
Wrong domains: _Admin.Global_!

Reply:
What is the use case for using Admin.Actions then? I'm lost here.

Reply:
Looking at current translations files, I'm right about the location of each translation key . This means translators will need to translate again these keys: do you agree?

Reply:
Hi @LouiseBonnard, domain names updated!